webv2/routes.go

```diff
@@ -46,6 +46,7 @@ func getRoutes() *mux.Router {
 	router.HandleFunc("/rename/fks", renameForeignKeys).Methods("POST")
 	router.HandleFunc("/rename/indexes", renameIndexes).Methods("POST")
 	router.HandleFunc("/add/indexes", addIndexes).Methods("POST")
+	router.HandleFunc("/update/indexes", updateIndexes).Methods("POST")
```

Comment:
can the rename and update apis not be merged ? I don't think its feasible to have multiple APIs with similar function.
cc: @bharadwaj-aditya

Reply:
Agreed. We should merge them if possible.

Reply:
Moved to backlog(no. 73) as it has dependencies of index id. Index id addition is in progress.